web/vital_records/forms.py

```diff
@@ -200,21 +210,20 @@ class OrderInfoForm(forms.ModelForm):
     )
     zip_code = forms.CharField(
         label="Zip code",
-        max_length=5,
+        max_length=10,
         required=True,
         widget=forms.TextInput(
             attrs={
                 "class": "form-control",
                 "inputmode": "numeric",
-                "minlength": "5",
-                "maxlength": "5",
+                "pattern": "[\d]{5}(-[\d]{4})?",
```

Comment:
This is from USDWS usability guidance: https://designsystem.digital.gov/templates/form-templates/address-form/#usability-guidance and recommended code: https://designsystem.digital.gov/templates/form-templates/address-form/#using-the-address-form-template-2